src/django/api/tests.py

```diff
@@ -131,6 +142,17 @@ def setUpTestData(cls):
             created_at=datetime(2022, 10, 2, hour=10, minute=5, tzinfo=timezone.utc),
```

Comment:
Not related to your changes, but found in testing. Would you mind adding `reviewed_at` to the the review object, a few lines above? We may not need both `created_at` and `reviewed_at`, but that's another discussion.

Reply:
Since we are adding a "Review Started" activity log for now, that object works well as a "review started but not yet submitted" test, so I left it as is for now.